CMakeLists.txt

```diff
@@ -149,7 +149,6 @@ function (llama_option_depr TYPE OLD NEW)
     endif()
 endfunction()
 
-llama_option_depr(FATAL_ERROR LLAMA_CUBLAS              GGML_CUDA)
```

Comment:
Why did you remove this? This is unrelated with your PR.
If you have a fatal error, just use the correct option.